revolv/templates/project/project.html

```diff
@@ -1,240 +1,319 @@
               <div class="row">
                 <div class="left-date">
-                  <span class="date-txt">{{ update.date|date:"o" }}</span>
-                  <span class="month-txt">{{ update.date|date:"M"|upper }}</span>
+                  <span class="date-txt">{{ update.date|date:"o" }}
+                  </span>
+                  <span class="month-txt">{{ update.date|date:"M"|upper }}
+                  </span>
                 </div>
                 <div class="main-info">
-                    {{ update.update_text|safe }}
-                    <ul class="social-ul">
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-fb icons"></i></a></li>
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-tw icons"></i></a></li>
-                      <li><a href="javascript:;" class="btn-icon-gray"><i class="icon-gg icons"></i></a></li>
-                    </ul>
+                  {{ update.update_text|safe }}
+                  <ul class="social-ul">
+                    <li>
+                      <a href="#" class="btn-icon-gray">
+                        <i class="icon-fb icons">
+                        </i>
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#" class="btn-icon-gray">
+                        <i class="icon-tw icons">
+                        </i>
+                      </a>
+                    </li>
+                    <li>
+                      <a href="#" class="btn-icon-gray">
+                        <i class="icon-gg icons">
+                        </i>
+                      </a>
+                    </li>
+                  </ul>
                 </div>
               </div>
               <!-- end .row -->
-              {% endfor %}
-            </div>
+            {% endfor %}
+          </div>
         {% endif %}
         <!-- end .list-area -->
         <div class="venue-area">
-          <h2 class="title-blue-border title-venue pull-right">VENUE</h2>
-          <div class="clearfix"></div>
+          <h2 class="title-blue-border title-venue pull-left">VENUE
+          </h2>
+          <div class="clearfix">
+          </div>
           <div class="venue-map" id="map-canvas">
           </div>
           <div class="venue-bottom">
-            <h4 class="title-h4">{{ project.title }}</h4>
+            <h4 class="title-h4">{{ project.title }}
+            </h4>
             <p>
-              <span>{{ project.location_street }}</span>
-              <span>{{ project.location_city_state_zip }}</span>
+              <span>{{ project.location_street }}
+              </span>
+              <span>{{ project.location_city_state_zip }}
+              </span>
             </p>
           </div>
         </div>
         <!-- end .venue-area -->
       </div>
-      <!-- end .main-area -->
-      <aside class="right-aside">
-        {% if project_donation_levels %}
```

Comment:
You need to use these template tags above in the donation levels section. This says:

1. If the project has donation levels (for example, $20, $40, or $80 suggested donation), then:
2. For each donation level:
3. Render the information (for example `<p>{{ donation_level.description }}</p>`) about each `donation_level`.